components/cartbox/allergens.blade.php

```diff
@@ -0,0 +1,17 @@
+@foreach ($menuItem->allergens->where('status', 1) ?? [] as $allergen)
+    <a
+        class="badge {{ !($hasMedia = $allergen->hasMedia('thumb')) ? 'badge-light' : '' }} rounded mt-2 mr-1"
+        data-toggle="tooltip"
+        title="{{ $allergen->name }}: {{ $allergen->description }}"
+    >
+        @if ($hasMedia)
+            <img
+                class="img-responsive img-rounded"
+                alt="{{ $allergen->name }}"
+                src="{{ $allergen->getThumb(['width' => $menuAllergenImageWidth, 'height' => $menuAllergenImageHeight]) }}"
+            >
+        @else
+            {{ $allergen->name }}
+        @endif
+    </a>
+@endforeach
```

Comment:
I don't see the point displaying the same allergens as on the menu items page, shouldn't this display the allergens for the menu item options instead?